common/utils/loadUserData.js

```diff
@@ -108,8 +108,9 @@ export async function loadUserData(api, store, alerts) {
       USER_QUERY,
       {
         id: userId,
-        activityAfter: now() - DAY * DEFAULT_NB_DAYS_MISSIONS_HISTORY,
-        nbFirstMissions: DEFAULT_NB_FIRST_MISSIONS
+        activityAfter: jsToUnixTimestamp(
+          subMonths(new Date(), DEFAULT_MONTH_RANGE_HISTORY).getTime()
```

Comment:
We retrieve on login the missions of the 2 last months, same as the history by default.